api/server/server.go

```diff
@@ -624,6 +624,17 @@ func (s *Server) getContainersLogs(version version.Version, w http.ResponseWrite
 		closeNotifier = notifier.CloseNotify()
 	}
 
+	c, err := s.daemon.Get(vars["name"])
```

Comment:
This actually breaking some convention, which was invented by @duglin and @runcom